### DISCUSSION TITLE
【不具合？】PlantUML を多用するとプレビューに失敗する

### DISCUSSION THREAD

**Discussion by TETSURO1999** [increments/qiita-discussions#936] [要望・アイデア・不具合]
おそらく2024年12月9日前後からだと思うのですが，タイトルにあるように PlantUML を使った記事を書くと，プレビューに失敗するようになりました。PlantUML による図が一つだけだと問題ないのですが，二つ，三つと増えると失敗する確率が増え，四つ以上だとほぼ確実に失敗します。症状としては，PlantUML による図が増えると記事を執筆する際の「エディタとプレビュー」画面でのレスポンスが急激に遅くなり，「下書き保存」のボタンを押しても画面上にいわゆる「ぐるぐる」表示が出てしばらく待たされた後，何も表示されません。少なくとも12月7日以前は正常にプレビューできていたと思います。

**Comment by Umekawa** [comment]
@TETSURO1999

ご報告ありがとうございます。
ご不便をおかけしており、申し訳ございません。

不具合をこちらの方でも確認することができたので、修正を行う予定です。
しばしの間お待ちいただけると幸いです :bow:

**Comment by Umekawa** [comment]
@TETSURO1999
お待たせしました！

マークダウンからhtmlを生成する際に利用している、社内のPlantUMLのサーバーに問題が起きていたため修正を行いました。
ご確認いただければ幸いです。

**Reply by TETSURO1999** [reply]
@Umekawa 
確認しました。ありがとうございました。
